assets/sass/base.scss

```diff
@@ -12,6 +12,11 @@ body {
   background: $color-neutral--lightest;
   color: $color-neutral--darkest;
   font-family: $font-family-content;
+  height: 100%;
+
+  &.--sticky {
+    padding-top:220px;
```

Comment:
Do you take responsiveness into account? Will the header always be exactly 220px high?

Also try to avoid `px`-units if you can. As we reset the `font-size` on the `html`-element, you can use `22rem` here.